### ISSUE TITLE
Train new prior

### ISSUE THREAD

**Issue by albertma-evotec** [MolecularAI/Reinvent#9]
Is there a tutorial for training new prior? I have tried the create_model.json.
 I cannot see where I can define how many epochs to train over either any code of the training loop of the prior.

Thanks

**Comment by patronov** [comment]
Im currently putting together an example of how to pre-process raw data from ChEMBL and will upload a jupyter notebook of how to train a model from scratch.
Otherwise, create_model.json is only for creating a blank model with a vocabulary.
The actual training is done by using transfer learning but the number of epochs should be set to higher value and the learning rate should either be adaptive or should change exponentially.
I will try to finish the example in the next few days and will upload it to [Reinvent Community](https://github.com/MolecularAI/ReinventCommunity).

**Comment by albertma-evotec** [comment]
Thanks Atanas. Yea I figured out it is done by the transfer learning mode. I have trained a workable one but I am looking forward to see what settings you would suggest.

**Comment by patronov** [comment]
I have added some examples for purging a database and then training a model from scratch [here](https://github.com/MolecularAI/ReinventCommunity/tree/master/notebooks). Also added a few files to this repo. Should be working fine, but let us know if there is anything unclear of anything is breaking.
